source/custom/problemsetguide/pset-intro.ptx

```diff
@@ -162,10 +162,11 @@
             At the same, work to remove any unnecessary text that does not help with the readability. 
         </p>
 
-        <image xml:id="writingguide" source="../../../linear-algebra/assets/custom/Su-MathWriting.png" width="100%">
+        <image xml:id="writingguide" source="custom/Su-MathWriting.png" width="100%">
             <description>
                 <p>
-                    This is a short guide to writing mathematics well by Francis Su (Harvey Mudd College). 
+                    This is a short guide to writing mathematics well by Francis Su (Harvey Mudd College). A
+                    <dataurl source="custom/Su-MathWriting.pdf">PDF</dataurl> is also avaialble. 
                     Use it a checklist when writing your solutions. <alert>Almost every deduction we give for communication is a result of work that violates at least one of these principles</alert>.
                 </p>
             </description>
```

Comment:
I've added a PDF of this document that students can access. Note that this description is usually hidden behind a little "info" icon, so students may not read it; maybe consider moving it into a separate paragraph?

Reply:
thank you!